app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file? I don't think we are working with views on the API

Reply:
Not sure. We decided to leave it, until instructed otherwise.